### DISCUSSION TITLE
Werid output just simple command "v=vector3d(1,1,0); plot(v)" anyone know why?

### DISCUSSION THREAD

**Discussion by gxd123** [mtex-toolbox/mtex#1062] [Ask Anything] [answered]
Hello, I using the simple command. But get the wrong direction, like following image. **Werid output!!** Can anyone help me?  I have tried many times even using different version. I don't know where there is a problem.
Thanks
```
v = vector3d(1,1,0);
plot(v)
```
**My output**
![bandicam 2021-05-27 11-40-17-422](https://user-images.githubusercontent.com/33199098/119762487-661c1400-bee0-11eb-987f-1b5792eaba8d.jpg)

**Shouldn't it be this one?** 
![bandicam 2021-05-27 11-33-28-957](https://user-images.githubusercontent.com/33199098/119762233-e42beb00-bedf-11eb-98d1-9b52f3146acf.jpg)

**Comment by ralfHielscher** [comment, marked as answer]
use
```
plotx2east
```
at the beginning of your code.
Ralf.

**Reply by gxd123** [reply]
ok. Thanks
